data_generation/cvdb_data.py

```diff
@@ -4,8 +4,12 @@
 from data_generation.data_objects import *
 
 
-def convert_year(year):
+def convert_year(year, anonymize=True):
     year = int(year)
+    
+    if not anonymize:    
```

Comment:
why this?

Reply:
I tried this at some point a long time ago and figured it'd be good to keep just in case